staff/gextremera/JS/Calculation.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Escribe un programa que escriba en la consola todos los múltiplos de 23 menores que 500 
+ * y al final escribe la suma de todos ellos.
+ */
+
+
+function multiplesOf23(){
+    var listMultiplesOf23 = 0;
+    var sumMultiplesOf23 = 0;
+    for (var i = 23; i < 500; i++){
+        if(i%23 === 0){
+           listMultiplesOf23 += i + '\t';
+           console.log(listMultiplesOf23);
```

Comment:
Clean a bit this code. You don't need this `console.log` here
`return` values instead and `console.log` the values returned by the function